demisto_sdk/commands/common/tests/git_config_test.py

```diff
@@ -213,7 +213,7 @@ def test_search_gitlab_id_valid(self, mocker, requests_mock):
         requests_mock.get(url, json=gitlab_response)
         mocker.patch.object(Repo, 'remote', return_value=Urls(['https://code.pan.run/xsoar/content-internal-dist.git']))
         git_config = GitContentConfig()
-        assert git_config.gitlab_id == 3606
+        assert git_config.project_id == 3606
```

Comment:
please add a comment explaining what this number is and why we test it

Reply:
added